lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -1267,6 +1272,11 @@ sub create_ia :Chained('base') :PathPart('create') :Args() {
             type => "duck.co"
         };
 
+        my $maintainer = { 
+            duckco => $c->user->username,
+            github => $gh_id? $c->d->rs('GitHub::User')->find({github_id => $gh_id})->login : ''
```

Comment:
The `?` are usually surrounded by spaces to make them easier to see, e.g. `a ? b : c;`, like the colon.